test/hook.lua

```diff
@@ -1,6 +1,19 @@
+buf = ''
```

Comment:
you could do something like:
```
buf = ''
wwwroot = 'randomdir'
files = {'index.html', 'test.html', ... }
```

Or load the `wwwroot` value and the files in another script.